vnext/ReactUWP/Views/TextInputViewManager.cpp

```diff
@@ -302,10 +324,27 @@ void TextInputShadowNode::updateProperties(const folly::dynamic&& props)
     }
     else if (pair.first == "text")
     {
-      if (pair.second.isString())
-        textBox.Text(asHstring(pair.second));
-      else if (pair.second.isNull())
-        textBox.ClearValue(winrt::TextBox::TextProperty());
+      if (m_mostRecentEventCount == m_nativeEventCount)
+      {     
+        if (pair.second.isString())
+        {
+          auto oldValue = textBox.Text();
+          auto newValue = asHstring(pair.second);
+          if (oldValue != newValue)
+          {
+            textBox.Text(newValue);
+          }
+        }
+        else if (pair.second.isNull())
+          textBox.ClearValue(winrt::TextBox::TextProperty());
+      }
+    }
+    else if (pair.first == "mostRecentEventCount")
```

Comment:
>pair.first [](start = 13, length = 10)

You appear to be behind master, pair.first, pair.second are no longer used in here everything should be propertyName, propertyValue